content/intro-to-storybook/ember/en/data.md

```diff
@@ -0,0 +1,251 @@
+---
+title: 'Wire in data'
+tocTitle: 'Data'
+description: 'Learn how to wire in data to your UI component'
+---
+
+So far we created isolated stateless components –great for Storybook, but ultimately not useful until we give them some data in our app.
+
+This tutorial doesn’t focus on the particulars of building an app so we won’t dig into those details here. But we will take a moment to look at a common pattern for wiring in data with container components.
+
+## Container components
+
+Our `TaskList` component as currently written is “presentational” (see [this blog post](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)) in that it doesn’t talk to anything external to its own implementation. To get data into it, we need a “container”.
+
+This example uses [ember redux](https://ember-redux.com/), a predictable state management library for ember, to build a simple data model for our app. However, the pattern used here applies just as well to other data management libraries like [Apollo](https://github.com/ember-graphql/ember-apollo-client).
```

Comment:
```suggestion
This example uses [ember redux](https://ember-redux.com/), a predictable state management library for Ember, to build a simple data model for our app. However, the pattern used here applies just as well to other data management libraries like [Apollo](https://github.com/ember-graphql/ember-apollo-client).
```

Reply:
It was fixed with the latest commit. Going to mark it as resolved if you don't mind.